Directory.Packages.props

```diff
@@ -1,58 +1,58 @@
 <!-- All csproj package references should not include version numbers. The version numbers are set using this props file. -->
 <Project>
-    <ItemDefinitionGroup>
```

Comment:
@JoseRenan, are the line indent changes intentional? If not, can we undo them?

Reply:
Fixed!